python/foglamp/plugins/south/playback/readme.rst

```diff
@@ -0,0 +1,8 @@
+************************
+FogLAMP South playback
+************************
+
+This directory contains a South plugin that we would like to have a south plugin that would read a file and present it as if it was coming from a sensor.
+
+The file format should be a CSV file, with options to allow different formatting of that file and different presentation of the readings to FogLAMP.
+
```

Comment:
Could you add the configuration option detail that exists in the pull request into the readme as well please.